apis/v1alpha1/local_object_reference_types.go

```diff
@@ -110,11 +120,12 @@ type SecretsDefaultLocalObjectReference struct {
 	//   resource: foos
 	//   name: myfoo
 	//
-	// +optional
+	// +kubebuilder:validation:MinLength=1
```

Comment:
min can be removed since the field is being defaulted.

Reply:
Updated, thanks!